torch/testing/_internal/common_methods_invocations.py

```diff
@@ -15574,36 +15573,57 @@ def error_inputs_mean(op_info, device, **kwargs):
                    ref=lambda x: np.reciprocal(np.sqrt(x)),
                    domain=(0, None),
                    dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16),
-                   dtypesIfCUDA=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
+                   dtypesIfCUDA=all_types_and_complex_and(torch.chalf, torch.bool, torch.half, torch.bfloat16),
                    decorators=(precisionOverride({torch.half: 5e-2}),),
                    assert_autodiffed=True,
                    supports_forward_ad=True,
                    supports_fwgrad_bwgrad=True,
                    skips=(
                        DecorateInfo(unittest.skip("Skipped!"), 'TestUnaryUfuncs', 'test_reference_numerics_extremal',
                                     dtypes=(torch.cfloat, torch.cdouble)),
+                       # AssertionError: Tensor-likes are not close!
```

Comment:
Can you share more detail as to why this is failing? (and update the comment accordingly)

Thanks!

Reply:
@kshitij12345, these tests failed due to tolerance reasons. Please take a look below to see complete failure:

<details>
<summary>Click Please</summary>

<p>

```python
FAIL [0.005s]: test_reference_numerics_large_rsqrt_cuda_complex32 (__main__.TestUnaryUfuncsCUDA)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/torch/testing/_internal/common_utils.py", line 1808, in wrapper
    method(*args, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/torch/testing/_internal/common_device_type.py", line 377, in instantiated_test
    result = test(self, **param_kwargs)
  File "/opt/conda/lib/python3.7/site-packages/torch/testing/_internal/common_device_type.py", line 784, in test_wrapper
    return test(*args, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/torch/testing/_internal/common_utils.py", line 1158, in wrapper
    fn(*args, **kwargs)
  File "test_unary_ufuncs.py", line 344, in test_reference_numerics_large
    self._test_reference_numerics(dtype, op, tensors)
  File "test_unary_ufuncs.py", line 302, in _test_reference_numerics
    expected, actual, msg, exact_dtype, equal_nan
  File "test_unary_ufuncs.py", line 273, in _helper_reference_numerics
    exact_dtype=exact_dtype,
  File "test_unary_ufuncs.py", line 226, in assertEqualHelper
    **kwargs
  File "/opt/conda/lib/python3.7/site-packages/torch/testing/_internal/common_utils.py", line 2258, in assertEqual
    msg=msg,
  File "/opt/conda/lib/python3.7/site-packages/torch/testing/_comparison.py", line 1086, in assert_equal
    raise error_metas[0].to_error()
AssertionError: Tensor-likes are not close!
Mismatched elements: 2000 / 10000 (20.0%)
Greatest absolute difference: nan at index (700,) (up to 0.01 allowed)
Greatest relative difference: nan at index (700,) (up to 0.001 allowed)
```

</p>
</details>

Also, I've updated the comment. Thanks!